utility/utils.py

```diff
@@ -754,3 +756,129 @@ def download_openshift_installer(version=defaults.INSTALLER_VERSION):
         run_cmd(f"tar xzvf {tarball}")
 
     return installer_filename
+
+
+def run_oc_cmd(cmd):
```

Comment:
redundant?

Reply:
nope,
run_cmd is for running any command
run_oc_cmd is for running oc and taking care of munchify etc.

Reply:
@RazTamir I meant if it's a duplicate with exec_oc_cmd() from ocp.py

Reply:
nice catch!
removed from utils.py